mep/settings.py

```diff
@@ -125,6 +154,18 @@
 
 STATIC_URL = '/static/'
 
+# These will be added to ``INSTALLED_APPS``, only if available.
+OPTIONAL_APPS = (
+    "debug_toolbar",
```

Comment:
I think this line, in concert with `urls.py` also loading `debug_toolbar` is causing djdt to be loaded twice.